dnsapi/dns_transip.sh

```diff
@@ -139,6 +141,17 @@ _transip_setup() {
   _saveaccountconf_mutable TRANSIP_Username "$TRANSIP_Username"
   _saveaccountconf_mutable TRANSIP_Key_File "$TRANSIP_Key_File"
 
+  # for automated testing, put 'base64 -w0 keyfile' output into github secret for TRANSIP_Key_File variable
+  if [ -n "$TEST_DNS" ]; then
```

Comment:
why do we need this ?

Reply:
Because the TransIP api requires an RSA key-file for authentication, but I didn't find a way to add a file to the secrets.
Also couldn't find how others before me did that with the CI, so chose this way to create a key file from the github secrets.

Reply:
upload your key file to somewhere, like:  https://transfer.sh/

And set `TRANSIP_Key_File=https://xxxx.download.url/xxxxxx`

when you use `TRANSIP_Key_File`,  check if it exists as a local file first.
otherwise, check if it starts with  "https://...", then just `get()` it and use it.